rosidl_generator_py/rosidl_generator_py/__init__.py

```diff
@@ -21,6 +21,7 @@
 
 try:
     from .generate_py_impl import generate_py
+    assert generate_py
```

Comment:
Thanks for looking into this @ernestmc. I believe you can more simple add `  # noqa` to the end of line 23, rather than adding an assert. I think that would be a better way to work around the warning.